src/components/ui/Accordion/index.tsx

```diff
@@ -0,0 +1,61 @@
+'use client';
+
+import { forwardRef } from 'react';
+
+import * as AccordionPrimitive from '@radix-ui/react-accordion';
+import { ChevronDown } from 'lucide-react';
+
+import { cn } from '@/libs/utils';
+
+const Accordion = AccordionPrimitive.Root;
+
+const AccordionItem = forwardRef<
+  React.ElementRef<typeof AccordionPrimitive.Item>,
+  React.ComponentPropsWithoutRef<typeof AccordionPrimitive.Item>
```

Comment:
Reactクラスは使わないでください(reactから明示的にimportしてきてください)
```suggestion
  ElementRef<typeof AccordionPrimitive.Item>,
  ComponentPropsWithoutRef<typeof AccordionPrimitive.Item>
```